Steamworks/src/org/usfirst/frc/team4786/robot/Robot.java

```diff
@@ -32,9 +33,8 @@
 	@Override
 	public void robotInit() {
 		oi = new OI();
-//		chooser.addDefault("Default Auto", new ExampleCommand());
+		chooser.addObject("Test PID", new DriveToPosition(1));
```

Comment:
I agree with Noah. The `SendableChooser` should be removed until we actually need to use it.